.github/workflows/hashicorp.yml

```
@@ -57,6 +46,19 @@ on:
       signore-client-secret:
         description: 'signore Client Secret (SIGNORE_CLIENT_SECRET)'
         required: true
+      hc-releases-key-staging:
+        description: 'Key for hc-releases staging'
+        required: false
+      hc-releases-key-prod:
+        description: 'Key for hc-releases prod'
+        required: false
+      hc-releases-host-prod:
+        description: 'Host for hc-releases prod'
+        required: false
+      hc-releases-host-staging:
+        description: 'Host for hc-releases staging'
+        required: false
```

Should we mark these as required and update the descriptions with the typical organization/repository secret names? We'll also need to update the README and engineering-docs when things are ready. 😄 

```suggestion
      hc-releases-key-staging:
        description: 'Key for hc-releases staging (HC_RELEASES_KEY_STAGING)'
        required: true
      hc-releases-key-prod:
        description: 'Key for hc-releases prod (HC_RELEASES_KEY_PROD)'
        required: true
      hc-releases-host-prod:
        description: 'Host for hc-releases prod (HC_RELEASES_HOST_PROD)'
        required: true
      hc-releases-host-staging:
        description: 'Host for hc-releases staging (HC_RELEASES_HOST_STAGING)'
        required: true
```